codechain/constants.rs

```diff
@@ -14,5 +14,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pub const DEFAULT_CONFIG_PATH: &'static str = "codechain/config/presets/config.dev.toml";
+pub const DEFAULT_CONFIG_PATH: &'static str = DEVELOPMENT_CONFIG_PATH;
```

Comment:
Make it use `PRODUCTION_CONFIG_PATH` in the release build.

Reply:
I fixed it.